packages/block-library/src/post-navigation-link/edit.js

```diff
@@ -94,20 +95,28 @@ export default function PostNavigationLinkEdit( {
 	return (
 		<>
 			<InspectorControls>
-				<PanelBody>
-					<ToggleControl
-						__nextHasNoMarginBottom
+				<ToolsPanel label={ __( 'Settings' ) }>
```

Comment:
Let's add dropdownMenuProps to place the dropdown on the left side of the block sidebar ([Example](https://github.com/WordPress/gutenberg/blob/ad8bdfe512cfef2e54d30459bff945f4231f27de/packages/block-library/src/archives/edit.js#L38)).